checker/checker.ml

```diff
@@ -208,8 +208,7 @@ let usage () =
 open Type_errors
 
 let anomaly_string () = str "Anomaly: "
-let report () = (str "." ++ spc () ++ str "Please report" ++
-                 strbrk "at " ++ str Coq_config.wwwbugtracker ++ str ".")
+let report () = strbrk (str (". Please report at " ^ Coq_config.wwwbugtracker ^ "."))
```

Comment:
That's not going to work, `str` and `strbrk` are `: string -> Pp.t`.

Reply:
I would do `str ". Please report at" ++ spc () ++ str Coq_config.wwwbugtracker ++ str "."`, or replace the `spc ()` by a string space (then `++` can be replaced by string concatenation).

Reply:
The latter is almost what I have now, except I use `strbrk` instead of `str`.

Reply:
Oh sorry indeed there should be only one `strbrk`.